content/docs/general/overviewFlowchart.md

```diff
@@ -0,0 +1,55 @@
+---
+# Title, summary, and page position.
+linktitle: Exam Flow and Overview in ExamTools
+weight: 25
+featured: false
+draft: false
+icon: book-reader
+icon_pack: fas
+
+# Page metadata.
+title: Exam Flow and Overview in ExamTools
+date: "2018-09-09T00:00:00Z"
+type: book  # Do not modify.
+---
+
+<style>
+table th:first-of-type {
+    width: 11%;
+}
+table th:nth-of-type(2) {
+    width: 10%;
+}
+table th:nth-of-type(3) {
+    width: 10%;
+}
+table th:nth-of-type(4) {
+    width: 10%;
+}
+table th:nth-of-type(5) {
+    width: 10%;
+}
+table th:nth-of-type(6) {
+    width: 10%;
+}
+table th:nth-of-type(7) {
+    width: 10%;
+}
+table th:nth-of-type(8) {
+    width: 10%;
+}
+table th:nth-of-type(9) {
+    width: 10%;
+}
+table th:nth-of-type(10) {
+    width: 10%;
+}
+</style>
+
+Below is a table showing the role of each person in a typical ExamTools Exam Session.
```

Comment:
This table simply has too many columns (10) to accommodate that much text. I suggest transposing it since there are only three rows. This would give you three 33% columns which should be readable.

Reply:
I agree, kind of.  My issue is that Hugo seems to be leaving about 30% of the screen to the right of the table as whitespace for no apparent reason.  If the table expanded right it would (likely) work out.

![image](https://user-images.githubusercontent.com/68024347/102160424-6acd9e00-3e53-11eb-9118-527e3ccbac6c.png)